### PR TITLE
Here is the object based version

### PR DESCRIPTION
**Pull request by damienmarchal**

### CODE REVIEW

File: speedlib/cranes/faller2.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+if __name__ == "__main__":
+    
+   
+    ip_1 = "172.17.217.217"
+    ip_2 = "172.17.217.217"
+    
+    crane_1 = Faller()
+    crane_1.init(ip_1)
+    print(crane_1.get_battery())
+    print(crane_1.change_speed(crane_1.MotorChassis,-50))
```
Comment (damienmarchal) [inline speedlib/cranes/faller2.py]: Here, it is unclear to me why MotorChassis is an attribute of the instance and not an attribute of the class. 
Do MotorChassis can change between instance ?

File: speedlib/cranes/faller2.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+from pint import UnitRegistry
+
+timeout = 2.5
+
+class Faller:
+    
+    def __init__(self):
+        
+        self.ureg = UnitRegistry()
+        self.MotorSpreader = 1
```
Comment (damienmarchal) [inline speedlib/cranes/faller2.py]: The following should be move out of the class

File: speedlib/cranes/faller2.py (hunk 1 above)
Comment (guillep) [reply speedlib/cranes/faller2.py]: Isn't it the same for all the other constants?

```python
       self.MotorSpreader = 1
       self.MotorCrab = 2
       self.MotorChassis = 3
       self.MotorDirectionBackward = 1
       self.MotorDirectionForward = 2
```

File: speedlib/cranes/faller2.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+import requests
+import re
+import time
+import asyncio
+from pint import UnitRegistry
+
+timeout = 2.5
+
+class Faller:
```
Comment (damienmarchal) [inline speedlib/cranes/faller2.py]: It may be clearer to give a more descriptive naming like:
Crane

File: speedlib/cranes/faller2.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        self.MotorCrab = 2
+        self.MotorChassis = 3
+        self.MotorDirectionBackward = 1
+        self.MotorDirectionForward = 2
+        
+        
+
+
+    def get_motor_info_from_number(self, sNr):
+
```
Comment (damienmarchal) [inline speedlib/cranes/faller2.py]: Remove the spaces

File: speedlib/cranes/faller2.py (hunk 3 above)
Comment (damienmarchal) [reply speedlib/cranes/faller2.py]: There is no docstrings

**Comment by damienmarchal** [comment]
Hi @samFabrice,

There is confict in this PR so it cannot be merged. Can you please fix it.

**Pull request merged**